docs/getting_started.md

```diff
@@ -93,35 +88,48 @@ In this tutorial, we'll create an APB for a containerized [hello world applicati
 ### Using apb init
 Our first task is to create the skeleton for your app using the apb tool.  The command for this is simple:
 ```
-apb init my-apb
+apb init my-apb --org <docker-org>
 ```
 At this point you will see the following file structure
 ```
 my-apb/
 ├── apb.yml
 ├── Dockerfile
 ├── playbooks
+│   ├── bind.yml
+│   ├── deprovision.yml
+│   ├── provision.yml
+│   └── unbind.yml
 └── roles
+    ├── bind-my-apb
+    │   └── tasks
+    ├── deprovision-my-apb
+    │   └── tasks
+    ├── provision-my-apb
+    │   └── tasks
+    └── unbind-my-apb
+        └── tasks
 ```
 Two files were created at the root directory, an `apb.yml` and a `Dockerfile`.  These are the minimum required for any APB.  For more information, visit the [design doc](https://github.com/fusor/ansible-playbook-bundle/blob/master/docs/design.md).
 ```yaml
 # apb.yml
 name: my-apb
 image: <docker-org>/my-apb
 description: "My New APB"
-bindable: false
+bindable: true
 async: optional
+parameters: []
 ```
 ```dockerfile
 # Dockerfile
 FROM ansibleplaybookbundle/apb-base
+# MAINTAINER {{ $MAINTAINER }}
```

Comment:
This is just a paste showing what `apb init` is generating.  I'd rather not change it, and it was just extra info if I make the user change it.